command/agent/command.go

```diff
@@ -1564,6 +1568,14 @@ Consul Options:
   -consul-client-http-check-name=<name>
     Specifies the HTTP health check name in Consul for the Nomad clients.
 
+  -consul-client-failures-before-critical
+    Specifies the number of consecutive failures before the Consul health check
+    is critical. Defaults to 0.
+
+  -consul-client-failures-before-warning
+    Specifies the number of consecutive failures before the Consul health check
+    shows a warning. Defaults to 0.
```

Comment:
Similar to the `-consul-client-http-check-name` field above, we should make it clear in the docstring that this is the Consul health check for the Nomad client only. (Because Consul itself has a client and that just makes the flag slightly ambiguous.)